tools/topology/topology2/cavs-sdw.conf

```diff
@@ -53,7 +53,7 @@ Define {
 	DEEP_BUFFER_PIPELINE_ID         15
 	DEEP_BUFFER_PCM_ID              31
 	DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
-	DEEP_BUFFER_PIPELINE_SINK       'mixout.2.1'
```

Comment:
I dont think we use fixup in the SOF repo but you can add a fixes tag @bardliao

Reply:
> I dont think we use fixup in the SOF repo but you can add a fixes tag @bardliao

Udpated